react-client/src/App.js

```diff
@@ -10,13 +10,21 @@ export default function App() {
   return (
     <Router>
       <div className="App">
-        <nav>
-          <li>
-            <Link to="/">Home</Link>
-          </li>
-          <li>
-            <Link to="/createTutorial">Create Tutorial</Link>
-          </li>
+        <nav
+          className="container-fluid d-flex justify-content-between align-items-center p-4 bg-primary"
+          style={{ height: "50px" }}
```

Comment:
Would probably put height on a separate class rather than bind directly to style, kinda keeping in line with my comment above about inconsistencies

Reply:
You're right. Will make a more consistent CSS styling.